websites/N/Ngewibu/presence.ts

```diff
@@ -4,9 +4,9 @@ const presence = new Presence({
 	browsingTimestamp = Math.floor(Date.now() / 1000);
 enum Assets {
 	Logo = "https://i.imgur.com/ZmylEPM.png",
-	Play = "https://i.imgur.com/6s4WyWY.png",
-	Pause = "https://i.imgur.com/PrYtpQb.png",
-	Search = "https://i.imgur.com/08wjeL0.png",
+	Play = "https://i.imgur.com/Am7HASZ.png",
+	Pause = "https://i.imgur.com/OKNf81I.png",
+	Search = "https://i.imgur.com/HNy8ilN.png",
```

Comment:
May i remake the small assets it's actually clunky on light theme discord

Before
![Screenshot_733](https://user-images.githubusercontent.com/12372481/236829239-706fd712-fcd7-4daa-b9e6-b525f18e5974.png)

After
![Screenshot_734](https://user-images.githubusercontent.com/12372481/236829323-40e27316-fce2-4858-8852-625977f30255.png)

Reply:
Will wait for https://github.com/PreMiD/Presences/pull/7345 I guess

Reply:
In my opinion we need no change the white base color, just adding some stroke with dark (#36393F) color here.
![download (3)](https://github.com/PreMiD/Presences/assets/12372481/992c63e7-fcc1-4b27-854d-4bb9acf59ab5)
So whenever It's on DC dark it's still white, otherwise it's just adding some additional dark on stroke

You can assign me to remake all of https://github.com/PreMiD/Presences/tree/main/.resources if this idea fit

Reply:
How would that look in the presence, cause I might prefer that too tbh, looks cleaner.

Reply:
@sinkaroid please update your Assets enum now, see index.d.ts for the ones that are already supported, you should be able to use them instantly without copying/importing.

Reply:
@Bas950 thanks for the follow up, looks like https://ngewibu.tv was updating their stuff, the thing is probably the template will changed again, I will draft this pr for now,

Once I get the fixed template from the maintainer I'll update this